### PR TITLE
doc: add CloudPilot AI v1.7.0 release doc

### PR DESCRIPTION
**Pull request by jwcesign**
doc: add v1.7.0 release doc

### CODE REVIEW

File: docs/changelog/changelog.mdx [hunk 1]
```diff
@@ -2,7 +2,21 @@
 title:  Changelog
 ---
 
-## October 2024 - V1.7.0 Comming Soon
+## November 2024 - V1.8.0
 
-### Feature
+### Feature:
 - Add support for aliyun cloud.
```
Comment (helen-frank) [inline docs/changelog/changelog.mdx:8]: maybe `alibaba cloud`

File: docs/changelog/changelog.mdx [hunk 2]
```diff
@@ -2,7 +2,26 @@
...
+
+✨✨✨ The most recent release brings some solid new features, enhancements, and bug fixes that should enhance your experience. Check out what's changed:
+
+### 🚀 New Features
+- **Control Minimum Spot Nodes**: Utilize the label `workload.cloudpilot.ai/spot-min-nodes` to specify the minimum number of spot nodes for pod scheduling within a deployment. This is optional and aims to prevent pods from becoming overly concentrated on a single spot node, with a default setting of 2 and a minimum value of 1.
+- **Immediate Node Template Changes**: Experience the benefits of immediate effects upon making changes to node templates, streamlining the optimization process.
+
+### ⚙️ Enhancements
+- **Error Reporting**: CloudPilot AI now displays corresponding errors when it is unable to calculate optimization results, providing clearer feedback on process interruptions.
+- **Automatic Target Optimization Node Architecture Configuration**: Enjoy the new feature that automatically configures the architecture of your target optimization node instances.
```
Comment (francis-jjk) [inline docs/changelog/changelog.mdx:21]: multi-arch(arm64 and amd64) awareness：When using multi-arch, we're defaulting to what you're using

File: docs/changelog/changelog.mdx [hunk 3]
```diff
@@ -2,7 +2,26 @@
...
+- Add support for alibaba cloud.
+
+
+## October 2024 - V1.7.0
+
+✨✨✨ The most recent release brings some solid new features, enhancements, and bug fixes that should enhance your experience. Check out what's changed:
+
+### 🚀 New Features
+- **Control Minimum Spot Nodes**: Utilize the label `workload.cloudpilot.ai/spot-min-nodes` to specify the minimum number of spot nodes for pod scheduling within a deployment. This is optional and aims to prevent pods from becoming overly concentrated on a single spot node, with a default setting of 2 and a minimum value of 1.
+- **Immediate Node Template Changes**: Experience the benefits of immediate effects upon making changes to node templates, streamlining the optimization process.
```
Comment (francis-jjk) [inline docs/changelog/changelog.mdx:17]: Node Template Changes Take Effect Instantly: Experience the benefits of immediate effects upon making changes to node templates, streamlining the optimization process.

**Comment by francis-jjk** [comment]
lgtm

**Comment by Aleegra** [comment]
lgtm ~

**Pull request merged**